pkg/pipelines/eventlisteners/eventlisteners.go

```diff
@@ -31,28 +31,28 @@ func Generate(githubRepo, ns, saName string) triggersv1.EventListener {
 					"dev-ci-build-from-pr",
 					devCIBuildFilters,
 					githubRepo,
-					"dev-ci-build-from-pr-binding",
+					"ci-build-from-pr-binding",
```

Comment:
How about we rename this to `github-pr-binding` ?

I think it indicates what it does a bit better (and implies that it's reusable).

Reply:
Renamed to `github-pr-binding`